source/virtualBuffers/__init__.py

```diff
@@ -412,13 +412,19 @@ def loadBuffer(self):
 
 	def _loadBuffer(self):
 		try:
+			if log.isEnabledFor(log.DEBUG):
+				startTime = time.time()
 			self.VBufHandle=NVDAHelper.localLib.VBuf_createBuffer(self.rootNVDAObject.appModule.helperLocalBindingHandle,self.rootDocHandle,self.rootID,unicode(self.backendName))
 			if not self.VBufHandle:
 				raise RuntimeError("Could not remotely create virtualBuffer")
 		except:
 			log.error("", exc_info=True)
 			queueHandler.queueFunction(queueHandler.eventQueue, self._loadBufferDone, success=False)
 			return
+		if log.isEnabledFor(log.DEBUG):
+			log.debug("Buffer load took %s sec, %d chars" % (

```

Comment:
This makes messages like:
Buffer load took 0.171000003815 sec, 273 chars
Maybe replace %s with %.3f?